packages/react/.babelrc

```diff
@@ -1,6 +1,6 @@
 {
   "exclude": "node_modules/**",
-  "plugins": [],
+  "plugins": ["@babel/plugin-proposal-class-properties"],
```

Comment:
Why is this needed?

Reply:
Eslint was complaining to some arrows when this wasn't there
Don't have a clear understanding why though...

Reply:
Weird, okay then 👍🏼